dapr/proto/runtime/v1/dapr.proto

```diff
@@ -68,6 +71,12 @@ service Dapr {
 
   // InvokeActor calls a method on an actor.
   rpc InvokeActor (InvokeActorRequest) returns (InvokeActorResponse) {}
+
+  // Gets metadata of the sidecar
+  rpc GetMetadata (google.protobuf.Empty) returns (GetMetadataResponse) {}
```

Comment:
Missing client implementation for these new APIs? Or is that going to be tackled in a separate issue?

Reply:
Added new calls